suite/src/utils/metrics.ts

```diff
@@ -0,0 +1,50 @@
+const { DiagConsoleLogger, DiagLogLevel, diag } = require('@opentelemetry/api');
```

Comment:
Did you mean to commit this file? Looks like it needs a few more updates. Would be great to have test metrics!

Reply:
This is a direct copy from the smoke test repo. Does it not serve any function? If not I am happy to remove it.

Reply:
Yeah, not sure why Val added this. Doesn't seem to be getting used 🤷🏼